File: Ajax/TemplateVSI/CommonFilesCS/Extender.cs
using System;
using System.Web.UI.WebControls;
using System.Web.UI;
using System.ComponentModel;
using System.ComponentModel.Design;
using AjaxControlToolkit;

[assembly: System.Web.UI.WebResource("$rootnamespace$.$baseitemname$Behavior.js", "text/javascript")]

namespace $rootnamespace$
{
    [Designer(typeof($baseitemname$Designer))]
    [ClientScriptResource("$rootnamespace$.$baseitemname$Behavior", "$rootnamespace$.$baseitemname$Behavior.js")]
    [TargetControlType(typeof(Control))]
    public class $baseitemname$Extender : ExtenderControlBase
    {
        // TODO: Add your property accessors here.
        //
        [ExtenderControlProperty]
        [DefaultValue("")]
        public string MyProperty
        {
            get
            {
                return GetPropertyValue("MyProperty", "");
            }
            set
            {
                SetPropertyValue("MyProperty", value);
            }
        }
    }
}

File: Ajax/TemplateVSI/CommonFilesVB/Extender.vb
Imports System
Imports System.ComponentModel
Imports System.Web.UI
Imports System.Web.UI.WebControls
Imports AjaxControlToolkit

#Region "Assembly Resource Attribute"
<Assembly: System.Web.UI.WebResource("$rootnamespace$.$baseitemname$Behavior.js", "text/javascript")> 
#End Region

Namespace $rootnamespace$

    <Designer(GetType($baseitemname$Designer))> _
    <ClientScriptResource("$rootnamespace$.$baseitemname$Behavior", "$rootnamespace$.$baseitemname$Behavior.js")> _
    <TargetControlType(GetType(Control))> _
    Public Class $baseitemname$Extender 
    Inherits ExtenderControlBase

    ' TODO: Add your property accessors here.
    <ExtenderControlProperty()> _
    <DefaultValue("")> _
    Public Property MyProperty() As String
        Get
            Return GetPropertyValue("MyProperty", "")
        End Get
        Set(ByVal value As String)
            SetPropertyValue("MyProperty", value)
        End Set
    End Property

    End Class

End Namespace
